src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/lambdaparametername/Example1.java

```diff
@@ -0,0 +1,19 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="LambdaParameterName"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.naming.lambdaparametername;
+
+import java.util.function.Function;
+
+// xdoc section -- start
+class Example1 {
+  Function<String, String> function1 = s -> s.toLowerCase();
+  Function<String, String> function2 =
+          S -> S.toLowerCase(); // violation, name 'S'
```

Comment:
Please keep comments as simple `// violation `
To match other Checks.
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/7d927bc_2023165810/checks/naming/localvariablename.html#Example1-code

https://github.com/checkstyle/checkstyle/blob/7d927bc9ce2cb9fcca250f186ef1645da8da5b08/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalVariableNameCheckExamplesTest.java#L28

Reply:
Sure, will change it